src/NHibernate.Test/CfgTest/ConfigurationSchemaFixture.cs

```diff
@@ -13,7 +11,7 @@ namespace NHibernate.Test.CfgTest
 	public class ConfigurationSchemaFixture
 	{
 		[Test]
-		public void InvalidConfig()
+		public void ThrowIfSessionFactoryIsMissing()
```

Comment:
Strictly speaking `<session-factory />` element is not required. It is required only when config is loaded not from app.config I guess.

Reply:
Yes, but the name is already long enough I find. `ThrowIfSessionFactoryIsMissingWhenNotFromAppConfig`?

Reply:
Yes, something like that. Or, better: `SessionFactoryIsRequiredWhenConfigurationIsLoadedNotFromAppConfig`